app/src/main/res/values-es/strings.xml

```diff
@@ -52,12 +63,12 @@
     <string name="illegal_state_transition">Mala transición de estado: %1$s</string>
 
     <string name="illegal_state_too_many_selections_when_permissions_granted">Tanto una sesión como una aplicación fueron seleccionadas cuando los permisos fueron concedidos.</string>
-    <string name="illegal_state_no_selections_when_permissions_granted">Ni una sesión ni una aplicación fueron seleccionadas cuando los permisos fueron concedidos.</string>
+    <string name="illegal_state_no_selections_when_permissions_granted">Ninguna sesión ó aplicación fueron seleccionadas cuando los permisos fueron concedidos.</string>
 
     <string name="illegal_state_no_filesystem_selected_when_credentials_selected">Las credenciales fueron guardadas para un sistema de archivos, pero ningún sistema de archivos fue seleccionado.</string>
     <string name="illegal_state_no_app_selected_when_preference_submitted">Los ajustes de la Aplicación fueron guardados, pero ninguna app fue seleccionada.</string>
     <string name="illegal_state_no_app_selected_when_preparation_started">Intentando iniciar la preparación de la aplicación, pero ninguna app fue seleccionada.</string>
-    <string name="illegal_state_error_fetching_app_database_entries">No se pudo obtener acceso a la base de datos de las aplicaciones.</string> <!-- I'm not sure if this translation is correct. -->
+    <string name="illegal_state_error_fetching_app_database_entries">No se pudieron obtener las las entradas de las bases de datos de las aplicaciones.</string> <!-- I'm not sure if this translation is correct. --> <!-- What about this -->
```

Comment:
This translation seems good as far as I can tell. Could you remove the comments from this line? That would be this part:
`<!-- I'm not sure if this translation is correct. --> <!-- What about this -->`

Reply:
Also `las las` is probably not right.

Reply:
Done, let me know if you find anything else. Thank you